File: Python/ex6.py
# 6. Print the sum of values in an array

def sum_of_values(list):
    length = len(list)
    sum = 0
    for i in range (0, length):
        sum = sum + list[i]
    return sum

out = sum_of_values([1,2,3,4,5])
print(out)

File: JavaScript/ex6.js
// 6. Print the sum of values in array

const prompt = require ("prompt-sync")();

function sum_of_values(list){
    let sum = 0;
    for (let i of list){
        sum += i;
    }
    return sum;
}

out = sum_of_values([1, 2, 3, 4, 5])
console.log(out)